index.js

```diff
@@ -0,0 +1,565 @@
+const fs = require('fs');
+const hf = require('./helper_functions.js');
+const data = require('./data.js');
+const db = require('./database.js')
+
+const app = express();
+
+app.use(bodyParser.json());
+
+app.get('/', function (req, res) {
+  //res.send('hello');
+  fs.createReadStream(filepath + 'test.mp3').pipe(res);
+});
+
+app.get('/1', function (req, res) { fs.createReadStream(filepath + '1.mp3').pipe(res); });
+app.get('/2', function (req, res) { fs.createReadStream(filepath + '2.mp3').pipe(res); });
+/*app.get('/3', function (req, res) { fs.createReadStream(filepath + '3.mp3').pipe(res); });
+app.get('/4', function (req, res) { fs.createReadStream(filepath + '4.mp3').pipe(res); });
+app.get('/5', function (req, res) { fs.createReadStream(filepath + '5.mp3').pipe(res); });
+app.get('/6', function (req, res) { fs.createReadStream(filepath + '6.mp3').pipe(res); });
+app.get('/7', function (req, res) { fs.createReadStream(filepath + '7.mp3').pipe(res); });
+app.get('/8', function (req, res) { fs.createReadStream(filepath + '8.mp3').pipe(res); });*/
+app.get('/a', function (req, res) { fs.createReadStream(filepath + 'geburtstagstanz.mp3').pipe(res); });
+app.get('/b', function (req, res) { fs.createReadStream(filepath + 'zau_zaubert.mp3').pipe(res); });
+app.get('/c', function (req, res) { fs.createReadStream(filepath + 'schneller_teller.mp3').pipe(res); });
+
+app.post('/', function (req, res) {
+  let context = {
+    succeed: function (result) {
+      res.json(result);
+    },
+    fail: function (error) {
+      console.log(error);
+    }
+  };
+  let alexa = Alexa.handler(req.body, context);
+
+  alexa.registerHandlers(
+    handlers,
+    start_handlers,
+    lection_handlers,
+    music_handlers,
+    game_handlers
+  );
+
+  alexa.appId = appId;
+  alexa.execute();
+});
+
+let server = app.listen(process.env.PORT || 4567, function () {
+  let host = server.address().address;
+  let port = server.address().port;
+  console.log("Server listening at http://%s:%s", host, port);
+  console.log("");
+});
+
+const handlers = {
+  "LaunchRequest": function () {
+    db.addUser(this.event.session.user.userId);
+
+    // USE ONCE AT DATABASE CREATION
+    //hf.createAllLectures();
+
+    this.handler.state = "_START";
+    this.emitWithState("Start");
+  },
+  "Unhandled": function () {
+    //this.response.speak("Auf Wiedersehen!").audioPlayerStop();
+    //this.emit(":responseReady");
+    this.handler.state = "_START";
+    this.emitWithState("Start");
+  }
+};
+
+const start_handlers = Alexa.CreateStateHandler("_START", {
+  "Start": function () {
+    if(this.attributes["unhandled"]){
+      this.attributes["unhandled"] = false;
+      this.response.speak("Das hat leider nicht geklappt, versuchen wir es noch einmal. Möchtest du lernen, üben oder Lieder hören?")
+      .listen("Sage lernen, üben oder Lieder hören").audioPlayerStop();
+      this.emit(":responseReady");
+    } else {
+      this.attributes["unhandled"] = false;
```

Comment:
This is executed in all code paths and could be used just once.

Reply:
How can I achieve that?